### ISSUE TITLE
[FEATURE] If you can add an option for changing the app name and icon that would be awesome

### ISSUE THREAD

**Issue by kejoirk** [anonfaded/FadCam#83]
### Description
Briefly describe the feature you would like to see.

### Benefit
Explain how this feature will improve the app or user experience.

### Additional Context
Add any other context or references for the requested feature.

### Priority
- [ ] Low
- [ ] Medium
- [ ] High

**Comment by anonfaded** [comment]
Yeah this is already planned and will be implemented in future. Thanks
